desi.lbl.gov config change may have broken desiutil unit tests

when running pytest:
```
====================================================== FAILURES =======================================================
_____________________________________________ TestInstall.test_verify_url _____________________________________________

self = <desiutil.test.test_install.TestInstall testMethod=test_verify_url>

    def test_verify_url(self):
        """Test the check for a valid svn URL.
        """
        options = self.desiInstall.get_options(['-v', 'desispec', '0.1'])
        out = self.desiInstall.get_product_version()
        url = self.desiInstall.identify_branch()
        self.assertTrue(self.desiInstall.verify_url())
        self.desiInstall.product_url = 'http://desi.lbl.gov/no/such/place'
        with self.assertRaises(DesiInstallException) as cm:
>           self.desiInstall.verify_url()
E           AssertionError: DesiInstallException not raised

py/desiutil/test/test_install.py:205: AssertionError
```

This failure is new after the desi.lbl.gov reconfig today; the test passed in the nightly integration tests ~3am PDT on July 20.

I initially discovered it while testing a new conda environment at NERSC, then verified that it also fails on my laptop, and it failed in the standard desi master environment cronjob tests last night.  e.g.
```
source /global/cfs/cdirs/desi/software/desi_environment.sh master
cd $DESIUTIL
pytest py/desiutil/test/test_install.py
...
        self.desiInstall.product_url = 'http://desi.lbl.gov/no/such/place'
        with self.assertRaises(DesiInstallException) as cm:
>           self.desiInstall.verify_url()
E           AssertionError: DesiInstallException not raised

py/desiutil/test/test_install.py:205: AssertionError
```
I would expect that it would also fail on your local machine or your own checkout of desiutil at NERSC; I haven't found anywhere that it does work since the desi.lbl.gov Cloudflare changes yesterday.

--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@ __pycache__
 htmlcov
 .coverage
 py/desiutil/.cache
+.pytest_cache
 
 # Sphinx
 doc/api

--- a/py/desiutil/test/test_install.py
+++ b/py/desiutil/test/test_install.py
@@ -200,7 +200,7 @@ class TestInstall(unittest.TestCase):
         out = self.desiInstall.get_product_version()
         url = self.desiInstall.identify_branch()
         self.assertTrue(self.desiInstall.verify_url())
-        self.desiInstall.product_url = 'http://desi.lbl.gov/no/such/place'
+        self.desiInstall.product_url = 'https://desi.lbl.gov/no/such/place'
         with self.assertRaises(DesiInstallException) as cm:
             self.desiInstall.verify_url()
         message = ("Error {0:d} querying GitHub URL: {1}.".format(

